src/app/components/core/notifications/notifications.component.ts

```diff
@@ -370,6 +245,18 @@ export class NotificationsComponent implements OnInit, OnDestroy {
         });
   }
 
+  /**
+   * Open Notification Settings modal
+   */
+  openNotificationSettingsModal() {
+    this.childData = {
+      'isNotificationServiceAvailable': this.isNotificationServiceAvailable,
```

Comment:
IMO in an object key holds the information. key name  `isNotificationServiceAvailable -> notificationServiceAvailable`. remove questionable `is`  from all keys. Also there is no need of single quote as well.